### PR TITLE
vb/tiles fix website with ts

### PR DESCRIPTION
**Pull request by belom88**
I3S & 3DTiles applications work

### CODE REVIEW

File: modules/tiles/src/tileset/helpers/frame-state.ts [hunk 1]
```diff
@@ -4,16 +4,16 @@ import {Ellipsoid} from '@math.gl/geospatial';
...
+    direction: number[];
+    up: number[];
   };
   viewport: {[key: string]: any};
   height: number;
   cullingVolume: CullingVolume;
   frameNumber: number; // TODO: This can be the same between updates, what number is unique for between updates?
   sseDenominator: number; // Assumes fovy = 60 degrees
-}
+};
```
Comment (ibgreen) [inline modules/tiles/src/tileset/helpers/frame-state.ts:16]: Nit: this should not need a semicolon

File: modules/tiles/src/tileset/tile-3d.ts [hunk 2]
```diff
@@ -31,13 +31,13 @@ function defined(x) {
...
  */
- export type TileHeaderProps = {
+export type TileHeaderProps = {
   tileset: Tileset3D;
   header: Object;
   parentHeader: TileHeader;
   basePath: string;
   extendedId: string;
-}
+};
```
Comment (ibgreen) [inline modules/tiles/src/tileset/tile-3d.ts:40]: ditto

File: modules/tiles/src/tileset/tile-3d.ts [hunk 3]
```diff
@@ -120,7 +120,7 @@ export default class TileHeader {
     header: {[key: string]: any},
     parentHeader: TileHeader,
     basePath: string,
-    extendedId: string = ''
+    extendedId = ''
```
Comment (ibgreen) [inline modules/tiles/src/tileset/tile-3d.ts:123]: Why drop the type?

File: modules/tiles/src/tileset/tile-3d.ts [hunk 4]
```diff
@@ -199,8 +200,6 @@ export default class TileHeader {
 
     this._getPriority = this._getPriority.bind(this);
 
-    this.transform = new Matrix4();
```
Comment (ibgreen) [inline modules/tiles/src/tileset/tile-3d.ts:202]: Whoops, good catch

File: modules/tiles/src/tileset/tile-3d.ts [hunk 5]
```diff
@@ -316,7 +315,7 @@ export default class TileHeader {
 
     const requestToken = await this.tileset._requestScheduler.scheduleRequest(
       this.id,
-      this._getPriority
+      this._getPriority // eslint-disable-line @typescript-eslint/unbound-method
```
Comment (ibgreen) [inline modules/tiles/src/tileset/tile-3d.ts:318]: Just bind it?

File: modules/tiles/src/tileset/tileset-3d.ts [hunk 6]
```diff
@@ -68,7 +68,7 @@ export type Tileset3DProps = {
   loadTiles?: boolean;
   fetchOptions?: {[key: string]: any};
   basePath?: string;
-}
+};
```
Comment (ibgreen) [inline modules/tiles/src/tileset/tileset-3d.ts:71]: Not needed

File: modules/tiles/src/tileset/traversers/tileset-traverser.ts [hunk 7]
```diff
@@ -9,7 +9,7 @@ export type TilesetTraverserProps = {
   onTraversalEnd?: (frameState) => any;
   viewportTraversersMap?: {[key: string]: any};
   basePath?: string;
-}
+};
```
Comment (ibgreen) [inline modules/tiles/src/tileset/traversers/tileset-traverser.ts:12]: Not needed?

**Review by ibgreen — APPROVED** [review]

File: modules/tiles/src/tileset/helpers/frame-state.ts (hunk 1 above)
Comment (belom88) [reply modules/tiles/src/tileset/helpers/frame-state.ts:16]: It's all after `yarn lint fix`

File: modules/tiles/src/tileset/tile-3d.ts (hunk 3 above)
Comment (belom88) [reply modules/tiles/src/tileset/tile-3d.ts:123]: image

Comment (ibgreen) [reply modules/tiles/src/tileset/tile-3d.ts:123]: Good catch. I will try to disable that rule.

**Pull request merged**